js/modules/k6/experimental/fs/file.go

```diff
@@ -71,9 +77,10 @@ var _ io.Reader = (*file)(nil)
 //
 // When using SeekModeStart, the offset must be positive.
 // Negative offsets are allowed when using `SeekModeCurrent` or `SeekModeEnd`.
-func (f *file) Seek(offset int, whence SeekMode) (int, error) {
-	newOffset := f.offset
+func (f *file) Seek(offset int64, whence SeekMode) (int64, error) {
+	startingOffset := f.offset.Load()
```

Comment:
As mentioned in other places - this definitely still have a logical race even if the race detector will not complain as it is not a data one.

Here specifically two calls to `Seek` can race on reading the same value and then both changing that instead of being sequential.

We should probably fix this later along with the rest of the sequencing of the operations.